### DISCUSSION TITLE
Histogram widget

### DISCUSSION THREAD

**Discussion by vfresnaud** [pymeasure/pymeasure#887] [Ideas]
Hi,
We have cloned the GraphWidget to display an histogram of the parameters (see below with test on random gaussian number).
It allows to quickly observe the distribution of the parameter under test (mean and +/-3 sigmas)
![image](https://user-images.githubusercontent.com/51748347/229851665-49b39299-43f5-4970-9927-71a979997d87.png)

We have made it a little bit quick and dirty, but I'm sure you can handle it quick and clean :)
It will be a great addition to the PyMeasure environment.

BR,
VF

**Comment by CasperSchippers** [comment]
That is an interesting type of graph; It probably depends very much on the measurement types whether it is relevant, but I can image that is can be very useful for some measurements.

I'm not sure whether we have a sufficient time & manpower to start working on this; sharing how you did this would give this a head start, but even them I can't promise when (or maybe even if) we will be able to implement that.

However, feel free to open a PR!
